app/containers/InterventionPage/ShareFilter.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useMemo } from 'react';
+import { Col } from 'react-grid-system';
+
+import globalMessages from 'global/i18n/globalMessages';
+
+import Select from 'components/Select';
+import { IntlShape } from 'react-intl';
+import { SelectOption } from 'components/Select/types';
+
+type Props = {
+  formatMessage: IntlShape['formatMessage'];
+  onChange: (values: string) => void;
+  active: string;
```

Comment:
I would be great to enumerize these values 

```
export enum SharingFilter {
  ONLY_SHARED_BY_ME = 'onlySharedByMe',
  ...
}
```
then
```suggestion
  onChange: (values: SharingFilter) => void;
  active: SharingFilter;
```